loki/apps-v1.Deployment-query-scheduler.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: query-scheduler
+  labels:
+    app.kubernetes.io/name: loki
+    app.kubernetes.io/component: query-scheduler
+spec:
+  minReadySeconds: 10
+  replicas: 2
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: loki
+      app.kubernetes.io/component: query-scheduler
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: loki
+        app.kubernetes.io/component: query-scheduler
+    spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchLabels:
+                app.kubernetes.io/name: loki
+                app.kubernetes.io/component: query-scheduler
+            topologyKey: kubernetes.io/hostname
+      containers:
+      - name: query-scheduler
+        args:
+        - -config.expand-env
+        - -config.file=/etc/loki/config/config.yaml
+        - -log.level=debug
```

Comment:
This could be quite noisy, is this intentional? Perhaps info is a sane default here.

Reply:
> This could be quite noisy, is this intentional? Perhaps info is a sane default here.

We do already use debug elsewhere e.g.
https://github.com/BitGo/kustomize-loki/blob/479694cdfaec0f7b7ddab8a2b11a311e45001682/loki/apps-v1.Deployment-query-frontend.yaml#L35